### ISSUE TITLE
New release to pip

### ISSUE THREAD

**Issue by Letme** [readthedocs/sphinx_rtd_theme#821]
### Problem

Some warnings have been fixed, but there is no official release yet.

#### Expected Results

Pip install.

**Comment by Daltz333** [comment]
I second this, is this project stale?

**Comment by stsewd** [comment]
It's active, I think there are some open PRs that need to be on the next release.

**Comment by Letme** [comment]
What is the reason for those PRs to be included? 

I think we should release as is and then release again, when new PRs are added. It is not like we are running out of release numbers...

**Comment by polyzen** [comment]
https://github.com/readthedocs/sphinx_rtd_theme/issues/379

**Comment by jessetan** [comment]
I think the important PRs that are still open relate to changing the build process to use Webpack.
I'm unable to run the current build process so we can not easily package a new version.

**Comment by Letme** [comment]
You should automate release with something like `setuptools_scm` ?

It has been more than half of the year since last release. While with all due respect "some open PRs need to be on next release", there is a lot of stuff in master which could be used.

To be more constructive: Can you point to pull requests that need to be completed, so that we can finally get that release?

**Comment by return42** [comment]
Can't believe that the last version on PyPi is from feb. 2019 .. so many bugs have been fixed in the meantime.  Everyone who installs from PyPi will get the deprecation warning  #739.  In the meantime use.  Installing direct from repo  ends with:

    pip install git+https://github.com/readthedocs/sphinx_rtd_theme
    error: [Errno 2] No such file or directory: 'node_modules/.bin/webpack': 'node_modules/.bin/webpack'

Sorry, but this is a very sloppy release management.

**Comment by alexsuter** [comment]
What can we do to get it released? Does someone have contact with the maintainer of this repo?

**Comment by davidfischer** [comment]
> Sorry, but this is a very sloppy release management.

@return42 Please make sure to keep your posts constructive.

> Installing direct from repo ends with

This Sphinx theme is not guaranteed to work with a pip install from the git repo. This may be especially true after the webpack transition is complete as compiled static assets may be missing from the repository but may be present in the PyPI release bundles. See discussion here: https://github.com/readthedocs/sphinx_rtd_theme/issues/379

**Comment by davidfischer** [comment]
> What can we do to get it released? Does someone have contact with the maintainer of this repo?

The maintainers are active but very simply have other priorities. In fact, multiple maintainers have posted in this thread. The theme is in the middle of changing how static assets are compiled and that process needs to be completed before release. If you want to help, you can start by reviewing https://github.com/readthedocs/sphinx_rtd_theme/pull/808.

**Comment by return42** [comment]
> Please make sure to keep your posts constructive.

Don't take me wrong .. To being 'constructive' I tried to find a workaround (like the hundred of other users wasting their time), but there is no workaround (or?). The issue was fixed in april #728.  IMO we have a lot of unneeded discussions and also some unneeded issue entries here about this error message (#725 #739 #823 #821).

> The theme is in the middle of changing how static assets are compiled

And that's what release management is all about. I'm not familiar with the details of this project, normally there is a feature branch for such development and a mainline branch with bug fixes.

Even if this is no longer possible, the release planer are able to create a branch with mainline bug fixes .. which can be released as interim solution until #808 is finished.

ATM the user is left alone and digging around .. from user's POV this is sloppy release management.

> The maintainers are active but very simply have other priorities.

Hey, we are all in the same situation, working on non profit projects and parallel earning money in our real job. Time is rear ...

This project has a great feedback, which also sets it apart.  If say this or that is sloppy release management, I don't want to criticize this project nor any maintainer or contributor.

It is just a fact, nothing more.

**Comment by davidfischer** [comment]
Thank you for a more constructive post. Everyone who works on this project is well aware of release management best practices. The development that introduced webpack as well as the problem you mentioned was in fact done in a topic branch, was approved, and was merged. Sometimes bugs slip into master.

However, to be constructive means a few things. Firstly, you could file the problem you're having as an actual issue rather than complaining about it in a separate ticket about a separate problem. Another possibility is to contribute a fix for the problem. No need this time as I already did it in https://github.com/readthedocs/sphinx_rtd_theme/pull/826. Even if you don't do either of those things, there's probably better ways to get your point across.

**Comment by ssbarnea** [comment]
While risking to annoy the (busy) maintainers, what users observe is that https://pypi.org/project/sphinx-rtd-theme/#history reports last release being **February 2019**.

**Comment by agjohnson** [comment]
> reports last release being February 2019

The next person to complain about the theme release trajectory is getting assigned ownership on some issues :smile: 

I'm going to close this issue as I don't think there is any more value in the conversation here. The theme maintainers are aware that we haven't released in a while. The maintainers are in part volunteers and do not work on the theme full time, we're all human, and this is an open source project -- everyone here has equal power to contribute to this repository.

The next release will target fixes required for Sphinx 2.+ support and the HTML 5 writer, there have not been many changes otherwise since last release.

**Comment by Letme** [comment]
That is a cool concept, so lets make it easier for yoi. Some of us can dedicate some of our full time job to develop open source projects, but for sure we cannot make decisions indead of you. So the deal here is that you make decisions on issues you want implemented and label them, so that anyone can pick them up and implement them. Then group them in milestone so that we know when next release is due.

In most projects releasing to pip is as simple as tagging, and some of the fixes on master already solve deprecation warnings for sphinx 2.1+, which means release is due. Number of open issues about this topic is also increasing so lets organize a bit and then everyone can help. Deal?

**Comment by return42** [comment]
ping .. one year later (#728) .. you can see a release somewhere on the horizon?
